unit1/lesson14.rst

```diff
@@ -13,16 +13,21 @@ Goals
 Objectives
 ----------
 
-- Know what the DOM acroynym stands for
+- Know what the DOM acronym stands for
 - Understand what a Document Object Model is
 - Know how the DOM relates to HTML, CSS, and the browser
 - Differentiate between dynamic and static web pages
-- Describe how the DOM is used to make web pages dynamic
-- Understand why the DOM is written in JavaScript and lives in the browser
-- Know that JavaScript is an event driven programing language
-- Explain what an event is
-- Trigger and handle events
-- Understand event bubbling
-- Explain how the DOM uses events
-- Familair with common DOM objects and methods
+- Describe how JavaScript and the DOM are used to make web pages dynamic
 - Alter a web page using the DOM
+- Describe how the JavaScript environment of a browser is different from other JavaScript environments, such as Node.js 
```

Comment:
Does the HTML content cover how to include JS files in an HTML page? If not, let's add that objective here.

Reply:
It does not cover that. We can add that to the learning objectives here or since it is a small thing, maybe the lesson text itself. What do you think?

Reply:
The HTML chapter only briefly mentions JavaScript and HTML. https://github.com/LaunchCodeEducation/intro-to-professional-web-dev/blob/master/src/chapters/html/background.rst